### ISSUE TITLE
Fails to work on Canary 1.6.11

### ISSUE THREAD

**Issue by TrenchFroast** [DiscordInjections/DiscordInjections#70]
On the latest Discord Canary version (1.6.11), the injector fails to inject the DOM properly and raises an exception:
```js
Uncaught TypeError: Failed to execute 'observe' on 'MutationObserver': parameter 1 is not of type 'Node'.
    at StateWatcher.observe (/home/ghosty/misc/DiscordInjections/DomReady/StateWatcher.js:36:23)
    at new StateWatcher (/home/ghosty/misc/DiscordInjections/DomReady/StateWatcher.js:7:14)
    at Object.<anonymous> (/home/ghosty/misc/DiscordInjections/DomReady/index.js:44:19)
    at Object.<anonymous> (/home/ghosty/misc/DiscordInjections/DomReady/index.js:59:3)
```

**Comment by MDMCK10** [comment]
:b:roke for me too, same error, DI version is 3.3.0 and discord version is:
DiscordCanary
"1.6.11"

**Comment by TTtie** [comment]
The only known workaround is to CTRL+R.

**Comment by AsaPlaysMC** [comment]
@TTtie Thanks, just got started with DI and was about to ask in the DTinker server xd

**Comment by TrenchFroast** [comment]
@TTtie that did not work for me, but it does work in the PTB

**Comment by TTtie** [comment]
@TrenchFroast works here on dankdows 10 canary though

**Comment by TTtie** [comment]
(or if you prefer, windows)

**Comment by tilda** [comment]
Fixed in 72fb169, latest Canary with latest master - Ubuntu 16.04

**Comment by TrenchFroast** [comment]
danke schön
